build.gradle

```diff
@@ -54,6 +54,7 @@ ext {
     robolectricVersion = libs.versions.robolectric.get()
     protocVersion = libs.versions.protoc.get()
     javaliteVersion = libs.versions.javalite.get()
+    protobuffJavaUtilVersion = libs.versions.protobuffjavautil.get()
```

Comment:
nit: the library is called `protobuf` and not `protobuff`.

Applies throughout

Reply:
+1 please fix in the `libs.versions.toml` file and `transport-backend-cct.gradle` too

Reply:
Done